test/models/stacks_test.rb

```diff
@@ -390,7 +390,7 @@ def self.deliver(event, stack, payload)
 
     test "updating the stack emit a hook" do
       expect_hook(:stack, @stack, action: :updated, stack: @stack) do
-        @stack.update(repo_name: 'foo')
+        @stack.update(environment: 'foo')
```

Comment:
This test has been incorrect since the `repo_*` attributes were moved to the repository model - the stack instance was marked as dirty because of this issue, `repo_name` doesn't exist anymore!